### PR TITLE
Windows Dart SDK updater: Fall back to Invoke-WebRequest if downloading the Dart SDK via BITS fails

### PR DESCRIPTION
**Pull request by jnschulze**
Currently, the Dart SDK updater script for Windows uses `Start-BitsTransfer` which might fail due to a disabled BITS service etc.
We should provide some fallback mechanism using `Invoke-WebRequest`

### CODE REVIEW

**Comment by goderbauer** [comment]
That's a good idea.

* Do you happen to know which versions of PowerShell support `Invoke-WebRequest`?
* Is there a better way to test if a system supports `Start-BitsTransfer` than using a try-catch?

**Comment by jnschulze** [comment]
`Invoke-WebRequest` was introduced in PowerShell 3.0 (see https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/invoke-webrequest?view=powershell-6) which was first shipped with Windows 8.

Apparently there's no other approach than trial-and-error to check for working BITS.

File: bin/internal/update_dart_sdk.ps1 [hunk 1]
```diff
@@ -43,8 +43,15 @@ if (Test-Path $dartSdkPath) {
...
 $dartSdkZip = "$cachePath\$dartZipName"
-Import-Module BitsTransfer
-Start-BitsTransfer -Source $dartSdkUrl -Destination $dartSdkZip
+
+Try {
+    Import-Module BitsTransfer
+    Start-BitsTransfer -Source $dartSdkUrl -Destination $dartSdkZip
+}
+Catch {
+    Write-Host "Downloading the Dart SDK using BITS failed, falling back to WebRequest"
```
Comment (goderbauer) [inline bin/internal/update_dart_sdk.ps1]: Can we change this to:
`Downloading the Dart SDK using the BITS service failed, retrying with WebRequest...`?

**Review by goderbauer — APPROVED** [review]
LGTM

Will merge as soon as our build is green.

**Pull request merged**